src/include/homestore/meta_service.hpp

```diff
@@ -332,6 +337,8 @@ class MetaBlkService {
     uint64_t max_compress_memory_size() const;
     uint64_t init_compress_memory_size() const;
 
+    std::vector< meta_sub_type > topological_sort(std::unordered_map< meta_sub_type, std::vector< meta_sub_type > >&);
```

Comment:
if we have define meta_subtype_vec_t, please use it instead of std::vector< meta_sub_type > in the whole PR.

Reply:
this func has been moved to hs_utils, we will use the original type there